cnf-tests/testsuites/pkg/pods/pods.go

```diff
@@ -413,3 +413,17 @@ func DetectDefaultRouteInterface(cs *testclient.ClientSet, pod corev1.Pod) (stri
 	}
 	return "", fmt.Errorf("default route not present")
 }
+
+func GetStringEventsForPod(cs *testclient.ClientSet, pod *corev1.Pod) string {
```

Comment:
Please, use `corev1client.EventsGetter` as it's more portable.

(`correv1client "k8s.io/client-go/kubernetes/typed/core/v1"`)